client/assets/js/meter.js

```diff
@@ -154,34 +154,36 @@ function flipSwitch(){
 //++
 //++
 function knobRange(elem){
+  debugger;
     switch ($(elem).attr('class')) {
-      case "range-indicator long":
+      case "close-range-knob":
```

Comment:
Says close range here, but within the case sets the knobMode to 'long'

Reply:
that's because the click case is basing on which mode the knob is currently on so it switches to the next one which is close. example(touches know which is currently on "long" it switches to "Med" , taps on  knob currently on "med", knob goes to "short" . I hope this clarifies it.

Reply:
same case for all three all are intended.

Reply:
Ever since Dan told us that using .css or .stye on js to manipulate dom is dangerous since that goes directly into as an inline-style, I've been trying to manipulate them through classes.

Reply:
That explains it perfectly. Thank you!

Yeah, I know Dan is of that opinion. That said, in an app that you designed yourself (rather than as a huge team) you can account for the inline styles in the code just fine. I'm good either way for this, I just get confused by the class route, since you have to check a whole other file to see what is happening instead of it just being in the code.